projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_dag.py

```diff
@@ -22,7 +22,12 @@
 
 
 class MetaJobsDag:
-    def __init__(self, team_name: str, meta_config: MetaPluginConfiguration):
+    def __init__(
+        self,
+        team_name: str,
+        meta_config: MetaPluginConfiguration,
+        arguments: Dict = None,
```

Comment:
This looks wrong. 

Arguments must be per job and not for the whole Dag

Reply:
I got misled by the vdk flag --arguments thinking that if you pass an argument to a DAG job, it would propagate it to all the orchestrated jobs. Consequently, this was brought to the game so that when the runner invokes _vdk run <job> --arguments <args>_, the DAG would pass a default value to all the jobs if such is missing. If you think that such scenario won't he ever needed to be handled, I will remove this logic.